ydb/_topic_writer/topic_writer.py

```diff
@@ -218,14 +97,31 @@ def __init__(
         self.created_at = created_at
         self.data = data
 
+    def _get_data_bytes(self) -> bytes:
+        if isinstance(self.data, bytes):
+            return self.data
+        if isinstance(self.data, str):
+            return self.data.encode()
+
+        raise ValueError("Bad data type: %s", type(self.data))
```

Comment:
formating won't work here, you should use explicit .format or f-string or %-style.